docker/build.sh

```diff
@@ -37,41 +37,48 @@ REMOVE_CONTAINER="--rm"
 function usage {
   echo -e "\nUsage: `basename $0` [OPTIONS]" \
           "\nBuild the current local TinkerPop project in a Docker container." \
+          "\nBy default all modules are run. If you prefer to run a specific module then please" \
+          "\nenter it as an option. Selecting specific modules changes the build to include only" \
+          "\nthose selected modules as well as gremlin-server, gremlin-test and neo4j-gremlin (and" \
+          "\ntheir dependencies)."
           "\n\nOptions are:\n" \
-          "\n\t-t,  --tests              run standard test suite for modules except non-Java GLVs" \
-          "\n\t-i,  --integration-tests  run integration tests for modules except non-Java GLVs" \
+          "\n\t-t,  --tests              run standard test suite" \
+          "\n\t-i,  --integration-tests  run integration tests" \
           "\n\t-n,  --neo4j              include Neo4j" \
-          "\n\t-a,  --all                run tests for all non-Java GLVs and gremlin-console" \
-          "\n\t-go, --golang             run tests for Go GLV" \
-          "\n\t-py, --python             run tests for Python GLV" \
-          "\n\t-dn, --dotnet             run tests for Dotnet GLV" \
-          "\n\t-js, --javascript         run tests for JavaScript GLV" \
-          "\n\t-c,  --console            run tests for gremlin-console" \
-          "\n\t-m,  --minimal            build only the modules required to test GLVs (driver, server, test, console, neo4j)" \
+          "\n\t-go, --golang             include only gremlin-go, gremlin-server, gremlin-test and" \
+          "\n                            neo4j-gremlin and any other selected module option" \
+          "\n\t-py, --python             include only gremlin-python, gremlin-server, gremlin-test" \
+          "\n                            and neo4j-gremlin and any other selected module option" \
+          "\n\t-dn, --dotnet             include only gremlin-dotnet, gremlin-server, gremlin-test" \
+          "\n                            and neo4j-gremlin and any other selected module option" \
+          "\n\t-js, --javascript         include only gremlin-javascript, gremlin-server," \
+          "\n                            gremlin-test and neo4j-gremlin and any other selected module option" \
+          "\n\t-c,  --console            include only gremlin-console, gremlin-server, gremlin-test" \
+          "\n                            and neo4j-gremlin and any other selected module option" \
```

Comment:
I think for the `and any other selected module option` part is confusing when you add to the individual flags. It may be more clear to specify in the general description, that the flags for the GLVs can be appended to build together.